### DISCUSSION TITLE
source.ip converted to text issue

### DISCUSSION THREAD

**Discussion by johnununu** [Security-Onion-Solutions/securityonion#7504] [Unsupported Versions] [answered]
This was kind of / not really answered in: [#5302](https://github.com/Security-Onion-Solutions/securityonion/discussions/5302)

We have multiple custom indexes that do not start with so-.

Where exactly do I need to add ... (or add so-kibana-*, etc).?

Do I add it in the custom template or the custom ingest pipeline, or somewhere else?

**Comment by weslambert** [comment, marked as answer]
This will be defined in the index-pattern for the index template.

How you apply the configuration depends on what version of Security Onion you are running, as we've switched to composable templates in the latest release. It looks like you have another issue open for an issue upgrading, so it might be easier to proceed with answering that in a more thorough fashion there as we work through the errors.
